docs/projects/pulsar_adc/index.rst

```diff
@@ -152,21 +152,40 @@ PulSAR_ADC_FMC
 Configuration modes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For AD40xx/ADAQ40xx projects the AD40XX_ADAQ400X_N parameter defines the type
-of ADC (AD40xx or ADAQ40xx). By default is set to 1. Depending on the project,
-some hardware modifications need to be done on the board and/or make command:
+The Zedboard based Pulsar ADC project can be built to use either the FMC 
+connector or the Pmod JA connector. The FMC_N_PMOD parameter is used to select
+between the two options. By default it is set to 1 (FMC).
 
-In case of the **ADAQ40xx** project:
+The Pulsar project also supports more exotic configurations required for
+certain ADCs like the AD7944. These modes are selected using the SPI_OP_MODE 
+parameter.
```

Comment:
```suggestion
The Pulsar project supports different configurations required for certain
ADCs, like the AD7944. These modes are selected using the `SPI_OP_MODE` 
parameter.
```

Also, below this, I think it would be best if you could list all the possible values for the parameters and what they mean.
Otherwise one would need to enter system_project.tcl and check in the `if` statements to know the possible values.